src/nft/components/explore/Table.tsx

```diff
@@ -280,7 +282,7 @@ function LoadingTable({ headerGroups, visibleColumns, ...props }: LoadingTablePr
           ))}
         </thead>
         <tbody {...props}>
-          {[...Array(DEFAULT_TRENDING_TABLE_QUERY_AMOUNT)].map((_, index) => (
+          {Array.from({ length: DEFAULT_TRENDING_TABLE_QUERY_AMOUNT }, (_, i) => i + 1).map((_, index) => (
```

Comment:
what advantage does this syntax have over the current?

Reply:
you are right, there are no advantages, We can use `[...Array(DEFAULT_TRENDING_TABLE_QUERY_AMOUNT)].map((_, index)`